src/main/java/com/minecolonies/entity/ai/citizen/fisherman/EntityAIWorkFisherman.java

```diff
@@ -506,18 +520,17 @@ private void throwRod()
         if (!world.isRemote)
         {
             worker.faceBlock(job.getWater());
-            world.playSoundAtEntity(
-              worker, "random.bow", VOLUME,
-              (float) (FREQUENCY_BOUND_VALUE
-                         / (random.nextDouble()
-                              * (FREQUENCY_UPPER_LIMIT_DIVIDER
-                                   - FREQUENCY_LOWER_LIMIT_DIVIDER)
-                              + FREQUENCY_LOWER_LIMIT_DIVIDER)));
+            world.playSound((EntityPlayer) null,
+              this.worker.getPosition(),
+              SoundEvents.ENTITY_BOBBER_THROW,
+              SoundCategory.NEUTRAL,
+              0.5F,
+              0.4F / (this.world.rand.nextFloat() * 0.4F + 0.8F));
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use a "double" or "BigDecimal" instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2164)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 0.4F to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 0.4F to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 0.8F to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)